ha/util/consul_kv_store.py

```diff
@@ -15,17 +15,27 @@
 # about this software or licensing, please email opensource@seagate.com or
 # cortx-questions@seagate.com.
 
+from base64 import b64encode
 import consul
+from consul import ConsulException
+from consul.base import ClientError
+from requests.exceptions import RequestException
+from urllib3.exceptions import HTTPError
 import socket
 from ha.const import HA_DELIM
+from cortx.utils.log import Log
+from typing import Any, Dict, List, NamedTuple, Optional
+
+TxPutKV = NamedTuple('TxPutKV', [('key', str), ('value', str),
+                                 ('cas', Optional[Any])])
```

Comment:
Can you tell me the use case of 'cas'?

Reply:
don't know if I have copied this function is copied from Hare,
But what I understand from the document is if I need to replicate values somewhere like some DB, this value can be set with some index.

Reply:
ok